### PR TITLE
feat: animate outgoing reactions

### PR DESCRIPTION
**Pull request by juliahermak**
I added an animation for the reaction. There is a question for clarification @adamant-al . Do you like this type of animation? There are others on this site, you can see https://animate.style/#documentation. At the moment, the animation is also triggered on page refresh. Is it ok to keep that behaviour?

### CODE REVIEW

**Comment by bludnic** [comment]
[UI: Animate reactions (copied from iOS)](https://trello.com/c/gfexf6be/429-ui-animate-reactions-copied-from-ios)

**Comment by juliahermak** [comment]
@adamant-al  Please look again.

File: src/components/AChat/AChatReactions/AChatReactions.vue [hunk 1]
```diff
@@ -71,10 +72,19 @@ export default defineComponent({
       return list.sort((left, right) => left.timestamp - right.timestamp)
     })
 
+    const shouldAnimate = (reaction: NormalizedChatMessageTransaction) => {
```
Comment (adamant-al) [inline src/components/AChat/AChatReactions/AChatReactions.vue:75]: The trick with a transaction timestamp is not reliable

**Comment by bludnic** [comment]
[Update Web3 lib for Ethereum](https://trello.com/c/U9RxukKB/367-update-web3-lib-for-ethereum)

**Comment by adamant-al** [comment]
Any update here?

**Review by bludnic — APPROVED** [review]

**Comment by bludnic** [comment]
@adamant-al check now

**Comment by adamant-al** [comment]
Looks complicated and still doesn't work with income reactions.

https://github.com/Adamant-im/adamant-im/assets/33592982/6725b56b-fdfa-4c68-8041-d0c41916be28

**Comment by bludnic** [comment]
@adamant-al can't find an easy solution to animate incoming reactions. I created a separate task for that https://trello.com/c/zuIvK28d/458-animate-incoming-reaction

**Comment by bludnic** [comment]
Closed in favor of #649

**Pull request closed without merging**